test/test_artifacts/v1/jupyter-lsp.test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG COSMOS_IMAGE
+FROM $COSMOS_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import jupyter_lsp"]
```

Comment:
import validation is a good start, but typically we're trying to go deeper with validation for these test cases if possible. If you take a look at other packages' respective docker test files you'll find different approaches such as cloning OSS git repositories where unit test suites can be executed, or in other cases executing a set of jupyter notebook files

Reply:
I spoke with @just4brown about this. It's impractical to test these particular packages in other ways, but for packages like altair (#84) we can execute a sample notebook, for example.